server/historian/lerna-package-lock.json

```diff
@@ -6061,11 +6061,6 @@
 				"array-find-index": "^1.0.1"
 			}
 		},
```

Comment:
Why is package-lock and lerna-package-lock changing in this PR? Your package.json file is not changing